components/bintray-publish/bin/publish-hab.sh

```diff
@@ -141,7 +140,7 @@ _find_system_commands() {
     exit_with "We require jfrog to publish artifacts to Bintray; aborting" 1
   fi
 
-  if $(tar --version 2>&1 | grep -q 'GNU tar'); then
+  if tar --version 2>&1 | grep -q 'GNU tar'; then
```

Comment:
There are a number of instances of
```bash
if $(some-command); then
```
That should almost certainly be
```bash
if some-command; then
```
In this example we are ok because the `-q` ensures no output, but if that were removed, this code would break:
```
$ if $(tar --version 2>&1 | grep 'GNU tar'); then echo "yay tar"; fi
tar: invalid option -- '('
Try 'tar --help' or 'tar --usage' for more information.
```
Without the superfluous command substitution, the code is more robust:
```
$ if tar --version 2>&1 | grep 'GNU tar'; then echo "yay tar"; fi
tar (GNU tar) 1.29
yay tar
```
See https://github.com/koalaman/shellcheck/wiki/SC2091